packages/embark-contracts-manager/src/index.js

```diff
@@ -535,12 +535,12 @@ class ContractsManager {
       }
     ], function (err) {
       if (err) {
-        self.compileError = true;
+        self.compileError = true; // TODO: WUT
```

Comment:
Yeah it's weird, I have no idea why it was done like this. Maybe we should put the actual error message there? At least, the waterfall or whatever would have something useful to display